chaos_genius/connectors/clickhouse.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+from sqlalchemy import create_engine
+from sqlalchemy import text
+from .base_db import BaseDb
+from .connector_utils import merge_dataframe_chunks
+
+
+class ClickHouseDb(BaseDb):
+
+    __SQL_IDENTIFIER = "`"
+
+    @property
+    def sql_identifier(self):
+        """Used to quote any SQL identifier in case of it using special characters or keywords."""
+        return self.__SQL_IDENTIFIER
+
+    db_name = "ClickHouse"
+    test_db_query = "SELECT 1"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def get_db_uri(self):
+        db_info = self.ds_info
+        host = db_info.get("host")
+        port = int(db_info.get("port"))
+        username = db_info.get("username")
+        database = db_info.get("database")
+        password = db_info.get("password")
+        if not (host and port and username and password and database):
+            raise NotImplementedError(
+                "Database Credential not found for ClickHouse."
+            )
+        self.sqlalchemy_db_uri = (
+            f"clickhouse+http://{username}:{password}@{host}:{port}/{database}"
```

Comment:
Is there a specific reason we have used `http` over the `native` driver which this library recommends using: https://github.com/xzkostyan/clickhouse-sqlalchemy#usage?

Reply:
Hello @Samyak2 , When i used the native client there were some errors during the mapping of dataTypes, but we can try again with the native client and see what in its bag ?